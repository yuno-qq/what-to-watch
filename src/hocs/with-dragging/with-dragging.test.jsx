import React from "react"
import renderer from "react-test-renderer"
import {describe, test, expect} from "@jest/globals"

import withDragging from "./with-dragging.jsx"


describe(`withDragging() snapshots`, () => {
  test(`should render correctly`, () => {
    const MockComponentWrapped = withDragging(() => <div className="MOCK_COMPONENT"></div>)
    const tree = renderer.create(<MockComponentWrapped/>)

    expect(tree).toMatchSnapshot()
  })
})
