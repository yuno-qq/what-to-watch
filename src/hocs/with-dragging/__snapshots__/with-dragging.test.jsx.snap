// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withDragging() snapshots: should render correctly 1`] = `
<div
  className="MOCK_COMPONENT"
/>
`;
